apps/aecore/src/aec_pow_cuckoo.erl

```diff
@@ -293,13 +292,13 @@ pack_header_and_nonce(Hash, Nonce) when byte_size(Hash) == 32 ->
 %%   Prefix setting load path to command so that blake2b.so is found in priv/lib
 %% @end
 %%------------------------------------------------------------------------------
--spec export_ld_lib_path() -> list(string()).
-export_ld_lib_path() ->
+-spec ld_lib_env() -> string().
+ld_lib_env() ->
     LdPathVar = case os:type() of
                  {unix, darwin} -> "DYLD_LIBRARY_PATH";
                  {unix, _}      -> "LD_LIBRARY_PATH"
                 end,
-    ["export ", LdPathVar, "=../lib:$", LdPathVar, "; "].
+    "env " ++ LdPathVar ++ "=../lib:$" ++ LdPathVar.
```

Comment:
I see `env` has already been tested in the [common tests](https://github.com/aeternity/epoch/blob/ab3c2394b7cc1dd73487feaa427c15108958be00/apps/aecuckoo/test/aecuckoo_SUITE.erl#L26) - even if not via erlexec.